pkg/installer/installer.go

```diff
@@ -100,30 +102,35 @@ const (
 	UninstallEtcdOperatorYamlConfig           = "spec.uninstall.etcdOperatorYaml"
 	UninstallEtcdClusterYamlConfig            = "spec.uninstall.etcdClusterYaml"
 	UninstallResourceQuotaYamlConfig          = "spec.uninstall.resourceQuotaYaml"
+	InstallLocalPathProvisionerConfig         = "spec.installLocalPathProvisioner"
+	UinstallLocalPathProvisionerConfig        = "spec.uninstallLocalPathProvisioner"
```

Comment:
Single variable for these two
`IncludeLocalPathProvisionerConfig = spec.includeLocalPathProvisioner`

And we'll also need 
`InstallLocalPathProvisionerYamlConfig = spec.install.localPathProvisionerYamlConfig`
and
`UninstallLocalPathProvisionerYamlConfig = spec.uninstall.localPathProvisionerYamlConfig`

Reply:
Ah, I think I misunderstood how our viper configfiles worked, this should be fixed now